Sources/Sentry/SentryBacktrace.cpp

```diff
@@ -63,6 +63,7 @@ namespace profiling {
             }
         }
         std::uintptr_t current;
+        // ???: is this to avoid ever gathering a backtrace for the profiler's sampling thread?
```

Comment:
@indragiek I was wondering if we should just move all these types of checks up and out of here, to earlier callsites.